rest-service/manager_rest/amqp_manager.py

```diff
@@ -121,13 +122,14 @@ def create_tenant_vhost_and_user(self, tenant):
             self.VHOST_NAME_PATTERN.format(tenant.name)
         username = tenant.rabbitmq_username or \
             self.USERNAME_PATTERN.format(tenant.name)
+
+        # The password is being stored encrypted in the DB
         password = (
             tenant.rabbitmq_password or
-            AMQPManager._generate_user_password()
+            encrypt(AMQPManager._generate_user_password())
```

Comment:
Eh, I don't like this too much - encrypting just to decrypt a second later.
I understand that the encrypted version is necessary later on, but this doesn't look too elegant